### PR TITLE
Update logo and add text

### PR DESCRIPTION
**Pull request by fureigh**
Closes issue #157.

### CODE REVIEW

File: web/sites/default/config/system.theme.global.yml [hunk 1]
```diff
@@ -9,8 +9,8 @@ features:
   favicon: true
   node_user_picture: false
 logo:
-  path: 'public://nsf_logo.png'
+  path: ''
   url: ''
-  use_default: false
+  use_default: true
```
Comment (cmc333333) [inline web/sites/default/config/system.theme.global.yml]: Weird that this file uses bools while the `nsf.settings.yml` file uses 0/1. Bleh.

File: web/sites/default/config/nsf.settings.yml [hunk 2]
```diff
@@ -5,9 +5,10 @@ features:
   favicon: 0
 logo:
   use_default: 0
-  path: 'public://nsf_logo-orig_0.png'
+  path: 'themes/nsf/logo.png'
 favicon:
   use_default: 1
+  path: ''
```
Comment (cmc333333) [inline web/sites/default/config/nsf.settings.yml]: Created #188 to track the missing favicon.

File: web/sites/default/config/system.theme.global.yml (hunk 1 above)
Comment (fureigh) [reply web/sites/default/config/system.theme.global.yml]: Agreed. :-/

File: web/sites/default/config/nsf.settings.yml (hunk 2 above)
Comment (fureigh) [reply web/sites/default/config/nsf.settings.yml]: Thank you!

**Pull request merged**